packages/selleo-design-core/src/Button.tsx

```diff
@@ -1,9 +1,25 @@
 import { h } from "preact";
 
-export function Button() {
-  return <button class="inline-flex bg-orange-500">Button</button>;
+export function PrimaryButtonSmall() {
+  return <button class="inline-flex justify-center text-sm text-white font-extrabold leading-7 tracking-normal min-w-[124px] min-h-[32px] bg-brand-primary hover:bg-brand-primary-600 focus:bg-brand-primary-600 focus:outline focus:outline-offset-1 focus:outline-1 focus:outline-brand-primary-600 rounded disabled:bg-brand-primary-500 disabled:opacity-30">Text</button>;
```

Comment:
can you check how will it behave when we change from focus: to focus-visible to get rid of focus state when clicking on it?

Reply:
@michalgren what do you think?

Reply:
Changes to `:focus-visible` so it's accessible via `tab` but not visible after button click.